CONTRIBUTING.md

```diff
@@ -129,6 +129,9 @@ Before you submit your pull request consider the following guidelines:
     git rebase master -i
     git push origin my-fix-branch -f
     ```
+    
+
+WARNING. Forced push may delete all in-code comments made in your previous PR
```

Comment:
You are right, it should be "in your PR".

I would probably not advice force push at all.
I understand that it creates some duplication but losing people's comments is worse.